agent/pagebroker/posix_copy_engine.cpp

```
@@ -1,14 +1,117 @@
+}
+
+Path
+PartialPath(const Path& destination)
+{
+  Path partial = destination;
+  partial += ".pagebroker-partial";
+  return partial;
+}
+
+uintmax_t
+DirectorySize(const Path& path)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)